Doc/tools/.nitignore

```diff
@@ -78,7 +77,6 @@ Doc/library/__future__.rst
 Doc/library/_thread.rst
 Doc/library/abc.rst
 Doc/library/aifc.rst
-Doc/library/argparse.rst
 Doc/library/ast.rst
```

Comment:
```suggestion
Doc/library/argparse.rst
Doc/library/ast.rst
```

Un-gate on the argparse reference until we decide what to do about `add_parser()`

Reply:
If we implement Adam's approach, this wouldn't be nessesary and we would instead add the target in `conf.py`.